tests/network/localnet/liblocalnet.py

```diff
@@ -1,12 +1,57 @@
+from collections.abc import Generator
+from contextlib import contextmanager
+
+import tests.network.libs.nodenetworkconfigurationpolicy as libnncp
 from libs.net import netattachdef
-from libs.net.vmspec import add_network_interface, add_volume_disk
+from libs.net.traffic_generator import Client, Server
+from libs.net.vmspec import IP_ADDRESS, add_network_interface, add_volume_disk, lookup_iface_status
 from libs.vm.affinity import new_pod_anti_affinity
 from libs.vm.factory import base_vmspec, fedora_vm
 from libs.vm.spec import CloudInitNoCloud, Interface, Metadata, Multus, Network
 from libs.vm.vm import BaseVirtualMachine, cloudinitdisk_storage
 from tests.network.libs import cloudinit
+from tests.network.libs.nodenetworkconfigurationpolicy import Resource
+from utilities.constants import OVS_BRIDGE
```

Comment:
Please create something new, this has no meaning when reading it.
The constant name should represent what it is not to reflect the value itself.

I'm not even sure that in all places it means the same.

I won't insist on this for this PR, but it needs to come as a follow up.